src/App.tsx

```diff
@@ -14,11 +14,196 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+const lengths: number[] = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+interface State {
+  goods: string[];
+  isStarted: boolean;
+  isReversed: boolean;
+  sortBy: 'none' | 'name' | 'length';
```

Comment:
let's make it enum and use in code instead of raw strings
```
enum SortBy {
  none,
  name,
  length,
}
```

```suggestion
  sortBy: SortBy;
```